src/sass/components/_hover.scss

```diff
@@ -0,0 +1,10 @@
+.hover-duration {
```

Comment:
nazwa .hover zakłada że dotyczy :hover'a a z kodu tak nie wynika ;) 

może lepiej zapisać to jako @mixin np: 
```
%helper-class {}

@mixin hover ($class: "%helper-class") {
  &:hover {
    @if $class {
      @extend #{$class};
    }
    @content;
  }
}
```

Reply:
potem możesz to wykorzystać np:
```
.test {
  text-align: center;
}

%a {
  color: red;
}

.abc {
  @include hover(".test") {
    background: blue;
  };
  @include hover("%a");
  @include hover;
}
```

Reply:
Jeśli chodzi o plik _hover.scss to mój błąd jest on związany z innym taskie nr9 :)